app/assets/javascripts/ReactComponents/table_sorts.js

```diff
@@ -10,6 +10,10 @@ function compare_dates(a, b) {
   return compare_numeric_values(parse_date(a), parse_date(b));
 }
 
+function compare_gradebox(a, b) {
```

Comment:
Identifier 'compare_gradebox' is not in camel case.<br>'compare_gradebox' is defined but never used.